src/visualizer/styles.css

```diff
@@ -54,9 +54,11 @@
 
 .code {
   align-content: center;
+  border: 2px solid grey;
+  /*border: 2px solid grey;*/
```

Comment:
not sure why is there 57 and 58? my file doesn't contain it.

Reply:
Why are there two styles.css files for sorting visualizer?

Reply:
oh wait i also dont know i will delete the one outside